src/modalvolumeData_setup.py

```diff
@@ -0,0 +1,107 @@
+import modal
```

Comment:
nit: For the filename, lets keep it in accordance with pip standards, all small case.